proco/connection_statistics/models.py

```diff
@@ -127,6 +137,22 @@ def get_connectivity_status(self):
         else:
             return self.CONNECTIVITY_STATUSES.moderate
 
+    def get_coverage_statuses(self):
```

Comment:
function named as `get_smth` definitely should return something. it's not your case, so please rename to something. for example: `update_coverage_status`